Lib/site.py

```diff
@@ -393,6 +393,17 @@ def setcopyright():
 def sethelper():
     builtins.help = _sitebuiltins._Helper()
 
+def gethistoryfile():
+    """Check if the PYTHONHISTORY environment variable is set and define
+    it as the .python_history file.  If PYTHONHISTORY is not set, use the
+    default ~/.python_history file.
+    """
+    h = os.environ.get("PYTHONHISTORY")
+    if h:
```

Comment:
Should we distinguish the case where is set but empty?  (`if h is not None`)

Reply:
I don't think so. Did you have something in mind?